docs/articles_en/learn_openvino/openvino_samples/get_started_demos.rst

```diff
@@ -145,6 +145,14 @@ Instructions below show how to build sample applications with CMake. If you are
 
                build_samples_msvc.bat
 
+            .. note::
+
+               Sample can also be built with using PowerShell. To do this, run the below command in PowerShell:
+
+               .. code-block:: sh
+
+                  & <path-to-build-samples-folder>/build_samples.ps1
```

Comment:
maybe we can add tabs with CMD / PowerShell ? Like in installation section

Reply:
I agree, the tab option makes things shorter and more clear... 
Also, whenever applicable, let's stick to wording similar to this "You can also build Samples using Powershell. To do so, run the following command:"
And for line 156 - "system" is preferred to "machine," let's improve it too.

Reply:
Changes applied in https://github.com/openvinotoolkit/openvino/pull/22631/commits/8437014fe2d4c01bf6be32d36012b0ef162c1ac3 & https://github.com/openvinotoolkit/openvino/pull/22631/commits/ae5e30927c3893e920e36da223e3e7878bfcbe40